flags.go

```diff
@@ -53,10 +53,6 @@ func defineFlagVar() *FlagVar {
 		"Enabling KCP Watcher to reconcile Watcher CRs created by KCP run operators")
 	flag.StringVar(&flagVar.skrWatcherPath, "skr-watcher-path", "./skr-webhook",
 		"The path to the skr watcher chart.")
-	flag.StringVar(&flagVar.skrWebhookMemoryLimits, "skr-webhook-memory-limits", "200Mi",
```

Comment:
why were these flags removed? is it not relevant to configure anymore?

Reply:
I believe these flags are relevant for production and may need dynamic adjustments, so this is something that should be still settable without recompiling imo. Wdyt?

Reply:
I think you're right. I will reintroduce them then 👍